### PR TITLE
feat(precinct-scanner): Scanner Not Detected screen

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
Task: #1476 

When the precinct scanner is disconnected, show a screen to the user and
don't poll for scanner status. Hopefully this will be a useful reminder
and also prevent making hanging requests to the scan service (which is
non-responsive when the scanner is disconnected - though likely we'll
want to fix that as well).

## Demo Video or Screenshot

Screen Shot 2022-03-08 at 3 32 00 PM

## Testing Plan 
Added automated test, also manually tested.

## Checklist
- [x] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

### CODE REVIEW

File: frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
```
Comment (jonahkagan) [inline frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx:1]: this is just a copy of the card reader not detected screen with the text changed

File: frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { fontSizeTheme, Main, MainChild, Prose, Screen } from '@votingworks/ui';
+
+function doNothing() {
+  // do nothing
+}
+
+interface Props {
+  useEffectToggleLargeDisplay?: () => void;
```
Comment (carolinemodic) [inline frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx:9]: this is a VxMark thing I think so you can remove it

**Review by carolinemodic — APPROVED** [review]
Nice!

File: frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx (hunk 2 above)
Comment (jonahkagan) [reply frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx]: thanks!

**Pull request merged**